src/pages/PostEditDelete.jsx

```diff
@@ -28,8 +29,12 @@ export default function PostEdit() {
         title: currentPost.title || "",
         description: currentPost.description || "",
         category: currentPost.category || "",
-        location: currentPost.location || "",
-        photos: currentPost.photos || [],
+        location: currentPost.zip || "",
+        photos: (currentPost.images || []).map((img) => ({
+          type: "existing",
+          url: img.image_url,
+          public_id: img.public_id,
```

Comment:
Isn't it `id`? Deletion isn't working properly.

```suggestion
          public_id: img.id,
```